src/Microsoft.DocAsCode.MarkdigEngine/Validation/ValidationExtension.cs

```diff
@@ -25,6 +28,13 @@ public void Setup(MarkdownPipelineBuilder pipeline)
 
             pipeline.DocumentProcessed += document =>
             {
+                if (InclusionContext.IsInclude 
+                && (string.Equals(Path.GetExtension((RelativePath)InclusionContext.RootFile), ".yml", StringComparison.OrdinalIgnoreCase) 
```

Comment:
No need to convert to `(RelativePath)`? Or convert it to `string` as to what `Path.GetExtension` needs?

Reply:
Sure